examples/ios/objc/Draw/AppDelegate.m

```diff
@@ -0,0 +1,84 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2014 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#import "AppDelegate.h"
+#import <Realm/Realm.h>
+#import "DrawView.h"
+#import "Constants.h"
+
+@implementation AppDelegate
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+    NSString *ipAddress = kIPAddress;
+
+    application.applicationSupportsShakeToEdit = YES;
+    
+    // Setup Global Error Handler
+    RLMSyncErrorReportingBlock globalErrorHandler = ^(NSError *error, RLMSyncSession *session) {
```

Comment:
Why set to a local `globalErrorHandler` just to then set that to `[RLMSyncManager sharedManager].errorHandler`? Just assign directly to `[RLMSyncManager sharedManager].errorHandler`.